apps/daffio/src/app/docs/components/docs-list/docs-list.component.html

```diff
@@ -5,7 +5,7 @@
     </ng-template>
 
     <ng-template #daffTreeItemTpl let-node>
-      <a daffTreeItem [node]="node" [routerLink]="node.url">{{ node.title }}</a>
+      <a [daffTreeItem]="ROUTER_LINK_ACTIVE_CONFIG" [node]="node" [routerLink]="node.url">{{ node.title }}</a>
```

Comment:
From a review of the directive's docs:

> <a routerLink="/user/bob" routerLinkActive #rla="routerLinkActive">
>   Bob {{ rla.isActive ? '(already open)' : ''}}
> </a>

Can you do:

> <a [selected]="rla.isActive"  #rla="routerLinkActive" [node]="node" [routerLink]="node.url">{{ node.title }}</a>

Reply:
yes I did something like that